ci/jobs/satellite6-pulp.yaml

```diff
@@ -0,0 +1,176 @@
+
+            # Inject environment variables.
+            echo "Injecting environment variables..."
+            echo "KEY_NAME=${KEY_NAME}" > jenkins_openstack.properties
+            echo "INSTANCE_IP=${INSTANCE_IP}" >> jenkins_openstack.properties
+            echo "INSTANCE_ID=${INSTANCE_ID}" >> jenkins_openstack.properties
+            echo "SECURITY_GROUP_NAME=${SECURITY_GROUP_NAME}" >> jenkins_openstack.properties
+            echo "REMOTE_JENKINS_USERNAME=${REMOTE_JENKINS_USERNAME}" >> jenkins_openstack.properties
+            echo "REMOTE_JENKINS_API_TOKEN=${REMOTE_JENKINS_API_TOKEN}" >> jenkins_openstack.properties
+            echo "REMOTE_JENKINS_URL=${REMOTE_JENKINS_URL}" >> jenkins_openstack.properties
+        - inject:
+            properties-file: jenkins_openstack.properties
+        - shell: |
+            # Wait until the OpenStack instance can be connected by SSH sessions.
+            # The following trigger builders will depend on SSH connectivity.
+            set +e
+            while true; do
+                ssh -o ConnectTimeout=5 \
+                    -o StrictHostKeyChecking=no \
+                    cloud-user@${INSTANCE_IP} echo testing
+                if [[ $? = 0 ]]; then
+                    break
+                fi
+                echo "$INSTANCE_IP is not yet ready...";
+                sleep 5
+            done
+            set -e
+        - trigger-builds:
+            - project: "satellite6-pulp-trigger"
+              block: true
+              bool-parameters:
+                  - name:  SETUP_FAKE_MANIFEST_CERTIFICATE
+                    value: true
+              predefined-parameters: |
+                  TRIGGER_TYPE=installation
+                  SERVER_HOSTNAME=${INSTANCE_IP}
+                  DISTRIBUTION=DOWNSTREAM
+                  SATELLITE_VERSION=6.2
+                  SATELLITE_RELEASE=GA
+                  REMOTE_JENKINS_USERNAME=${REMOTE_JENKINS_USERNAME}
+                  REMOTE_JENKINS_API_TOKEN=${REMOTE_JENKINS_API_TOKEN}
+                  REMOTE_JENKINS_URL=${REMOTE_JENKINS_URL}
+        - trigger-builds:
+            - project: "satellite6-pulp-upgrade"
+              block: true
+              predefined-parameters: |
+                  INSTANCE_IP=${INSTANCE_IP}
+        - trigger-builds:
+            - project: "satellite6-pulp-trigger"
+              block: true
+              predefined-parameters: |
+                  TRIGGER_TYPE=robottelo
+                  SERVER_HOSTNAME=${INSTANCE_IP}
+                  TEST_TYPE=endtoend-api
+                  REMOTE_JENKINS_USERNAME=${REMOTE_JENKINS_USERNAME}
+                  REMOTE_JENKINS_API_TOKEN=${REMOTE_JENKINS_API_TOKEN}
+                  REMOTE_JENKINS_URL=${REMOTE_JENKINS_URL}
+        - shell: |
+            echo ${REMOTE_JENKINS_USERNAME}
+            source sat-pulp/bin/activate
+            source $OPENSTACK_CREDENTIALS
+            get_instance="nova list | grep ${INSTANCE_ID})"
```

Comment:
Trailing `)` left on the string. Also avoid single use variable. change get_instance on line below with its value.